hbase-server/src/main/java/org/apache/hadoop/hbase/security/access/AccessController.java

```diff
@@ -1848,13 +1848,6 @@ public void postScannerClose(final ObserverContext<RegionCoprocessorEnvironment>
     scannerOwners.remove(s);
   }
 
-  @Override
-  public boolean postScannerFilterRow(final ObserverContext<RegionCoprocessorEnvironment> e,
-      final InternalScanner s, final Cell curRowCell, final boolean hasMore) throws IOException {
-    // 'default' in RegionObserver might do unnecessary copy for Off heap backed Cells.
```

Comment:
Actually in RegionObserver we don't do any copy so this comment was irrelevant. So it is ok to remove the default impl in the security CP that we have with hbase.